### ISSUE TITLE
WCAG issues with the likert component

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#316]
### Description of the bug

There are a couple of WCAG issues with the "likert component" that should be fixed. 

## 1. Title element is a h5. 
We should avoid headers in our components, and if we need them they should probably be h2 by default and possible to edit by the app developer. As it is now it will in most cases cause the app to fail the [1.3.1. info and relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships) by jumping over some elements in the header order.
[Example of how header structure should be (Norwegian)](https://www.uutilsynet.no/veiledning/tekst-og-struktur/226#overskriftsniver)

![Skjermbilde 2022-07-04 kl  15 36 10](https://user-images.githubusercontent.com/33222679/177170525-f52d1350-6fa1-47b5-93cd-178eb7e24c4e.png)

## 2. Empty `<th>` element
There is an empty header element in the table. This is breaking [1.3.1. info and relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships)
![Skjermbilde 2022-07-04 kl  15 37 42](https://user-images.githubusercontent.com/33222679/177171317-fe6dfce5-5f42-45c9-9167-2b23fe3cf7d0.png)

## 3. Empty labels
The testing tool Wave is giving a warning that the component contain labels, but that they do noy have any content. I tested it with a screen reader and every bit of information in relation to the radio button was read. (Tested with VoiceOver) Maybe look into if the labels should be structured in another way? 

One weird bug was that the label was read twice by Voice Over
![Skjermbilde 2022-07-04 kl  15 25 19](https://user-images.githubusercontent.com/33222679/177172044-c5b3d176-93ec-40b1-985e-b6be0613af26.png)

### Steps To Reproduce

Go to an app with a likert component
Run wave or another testing tool

**Comment by Magnusrm** [comment]
Could be looked at together with #507

**Comment by mijohansen** [comment]
This issue is partly unsolveable with the current use of material-UI which  doesn't support a two-header setup that the likert component needs. The problem with empty headers is probably solveable as we could just provide a text there to indicate that the column underneath is a column of questions.

https://www.w3.org/WAI/tutorials/tables/two-headers/
